options.c

```diff
@@ -1318,6 +1319,21 @@ static void toggle_pause_on_output_change(void *data)
 	pause_on_output_change ^= 1;
 }
 
+static void get_block_key_paste(void *data, char *buf, size_t size)
+{
+	strscpy(buf, bool_names[block_key_paste], size);
+}
+
+static void set_block_key_paste(void *data, const char *buf)
+{
+	parse_bool(buf, &pause_on_output_change);
+}
+
+static void toggle_block_key_paste(void *data)
+{
+	pause_on_output_change ^= 1;
+}
```

Comment:
Incomplete copy-edit here and above, "pause_on_output_change".

Reply:
:facepalm: